frontend/src/components/Repository.vue

```diff
@@ -59,7 +59,8 @@
             </div>
           </div>
         </div>
-        <br />
+        <h4 id="integrated-models-history" class="title is-4">History of integrated models</h4>
+        <gem-history />
```

Comment:
I'd suggest adding some padding, either below `<gem-history>` or above the `<h3>` title below.

Reply:
I would also suggest some extra space, but between this `<h4>` and the `<div>` above it that relates to `Integrated models`

Reply:
I have added some margins here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/972/commits/84ec2526c74c87dee796d92d2160a306e0fedf9f

Please let me know if you think they should be adjusted. 
<img width="1564" alt="image" src="https://user-images.githubusercontent.com/423498/166910369-01130fa8-d50b-459b-ac06-8cbb296074f7.png">

Reply:
My suggestion is 1,5-2x the space above `GEM Repository` (as shown in the figure above).

Reply:
> increase the space between the chart and `GEM Repository`

Yes, by a lot :)

Reply:
Ok sure. FYI bulma runs out of helper numbers after `-6` (https://bulma.io/documentation/helpers/spacing-helpers/). The inconsistent scale also bothers me. @mihai-sysbio do you prefer using bulma helper for the top margin and custom css for the bottom margin? FWIW, I would use custom css for both in this case.